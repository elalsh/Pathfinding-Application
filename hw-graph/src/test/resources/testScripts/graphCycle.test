# Tests a cycle with 4 nodes in the graph

# Create a graph
CreateGraph graph1

# Add nodes A, B, C and D
AddNode graph1 A
AddNode graph1 B
AddNode graph1 C
AddNode graph1 D

# add an edges to the Graph
AddEdge graph1 A B e1
AddEdge graph1 B C e2
AddEdge graph1 C D e3
AddEdge graph1 D A e4

# Print the nodes in the graph and the outgoing edges from A, B, C, D
ListNodes graph1
ListChildren graph1 A
ListChildren graph1 B
ListChildren graph1 C
ListChildren graph1 D
