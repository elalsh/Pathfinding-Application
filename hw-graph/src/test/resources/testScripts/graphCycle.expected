# Tests a cycle with 4 nodes in the graph

# Create a graph
created graph graph1

# Add nodes A, B, C and D
added node A to graph1
added node B to graph1
added node C to graph1
added node D to graph1

# add an edges to the Graph
added edge e1 from A to B in graph1
added edge e2 from B to C in graph1
added edge e3 from C to D in graph1
added edge e4 from D to A in graph1

# Print the nodes in the graph and the outgoing edges from A, B, C, D
graph1 contains: A B C D
the children of A in graph1 are: B(e1)
the children of B in graph1 are: C(e2)
the children of C in graph1 are: D(e3)
the children of D in graph1 are: A(e4)
